doc/lightning-renepay.7.md

```diff
@@ -0,0 +1,149 @@
+lightning-renepay -- Command for sending a payment to a BOLT11 invoice
+==================================================================
+
+SYNOPSIS
+--------
+**(WARNING: experimental)**
```

Comment:
we have a flag `--experimental-renepay`?

Reply:
No we don't have that flag. The entire plugin is experimental.

Reply:
Worried that this will give confusion people because iirc our experimental stuff is under a single flag. But I think till the people do not start to complain, it is fine

If I was writing the docs, I replace the experimental field and add an alert that explains that we are experimenting with this plugin to improve the pathfinding and the pay user experience.

Just to communicate to the user that it would be great to test it and report bugs

Reply:
OK. I'll change that. Thanks.